Source/Core/Core/Lua/LuaFunctions/LuaBitFunctions.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <string>
+#include "src/lua.hpp"
```

Comment:
The includes for lua source and headers should be qualifed is `<Externals/Lua/src/lua.hpp>` (or some kind of name separation that makes it obvious it's an external), etc.

`src/...` is an extremely vague inclusion source and does have a chance to conflict. This is also at odds with how we also handle a most of our other external includes.